.circleci/config.yml

```diff
@@ -0,0 +1,306 @@
+version: 2
+
+references:
+  container_config: &container_config
+    docker:
+      - image: aetrnty/builder
+        user: builder
+    working_directory: ~/epoch
+
+  rebar_cache_key: &rebar_cache_key rebar-cache-{{ checksum "rebar.lock" }}-{{ checksum "rebar.config" }}
+  restore_rebar_cache: &restore_rebar_cache
+    restore_cache:
+      key: *rebar_cache_key
+
+  restore_build_cache: &restore_build_cache
+    restore_cache:
+      key: build-cache-v1
+
+  packages_workspace: &packages_workspace /tmp/packages
+  build_package: &build_package
+    run:
+      name: Build Package Tarball
+      environment:
+        PACKAGES_DIR: *packages_workspace
+      command: |
+        if [ "$(uname -s)" == "Darwin" ]; then
+            PKG_SUFFIX="osx-$(sw_vers -productVersion)"
+        elif [ "$(uname -s)" == "Linux" ]; then
+            PKG_SUFFIX="ubuntu"
+        fi
+        make prod-package
+        mkdir ${PACKAGES_DIR:?}
+        mv _build/prod/rel/epoch/epoch-$(cat VERSION).tar.gz ${PACKAGES_DIR:?}/epoch-$(cat VERSION)-${PKG_SUFFIX}.tar.gz
+
+  test_package: &test_package
```

Comment:
Does CircleCI support upload of package to GitHub release?

Reply:
I'm afraid not. We could do that manually until we find a simple tool that we can use to do that (I believe that couple of `curl` should also do.)